components/bl-autocomplete-component/src/helpers/validate.js

```diff
@@ -1,17 +1,7 @@
 export const validate = items => {
-  if (!items) {
+  if (!Array.isArray(items)) {
```

Comment:
Maybe we should add some error logs for this cases? Or at first render it could be empty even if we pass options?